packages/live-preview-sdk/src/index.ts

```diff
@@ -120,7 +124,11 @@ export class ContentfulLivePreview {
 
       // setup the live preview plugins (inspectorMode and liveUpdates)
       if (this.inspectorModeEnabled) {
-        this.inspectorMode = new InspectorMode({ locale, targetOrigin: this.targetOrigin });
+        this.inspectorMode = new InspectorMode({
```

Comment:
Not in this one, but do we have already a ticket to update documentation and create an example for auto-tagging?

Reply:
https://contentful.atlassian.net/browse/TOL-1784